src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -1196,6 +1199,18 @@ internal bool DeriveUseSiteInfoFromCustomModifiers(ref UseSiteInfo<AssemblySymbo
             return false;
         }
 
+#nullable enable
+        internal void DeriveUseSiteInfoFromCompilerFeatureRequiredAttributes(ref UseSiteInfo<AssemblySymbol> result, System.Reflection.Metadata.EntityHandle handle, CompilerFeatureRequiredFeatures allowedFeatures)
```

Comment:
>DeriveUseSiteInfoFromCompilerFeatureRequiredAttributes [](http://example.com/codeflow?start=22&length=54)

This is a PE specific helper. Consider moving it to PE namespace. For example it can be placed as a static method on PEModuleSymbol.